frontend/.editorconfig

```diff
@@ -0,0 +1,14 @@
+# Editor configuration, see https://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+insert_final_newline = true
+trim_trailing_whitespace = true
+quote_type = single
+
+[*.md]
+max_line_length = off
+trim_trailing_whitespace = false
```

Comment:
Please put this file in `.gitignore` (either local or global)

Reply:
I am not so sure about this. This file is provided in default angular project.
I understand your POV, that IDE-config files could be ignored, but why it is shipped with official angular app?

Reply:
I read more about this file and it's meant to be cross-IDE editor configuration. I think it can be thrown into the same sack with `tslint.json` because it also plays towards code style unification.
At this point, I think we can leave this file committed.